app/views/schools/PageSchools.vue

```diff
@@ -7,11 +7,11 @@
             <mixed-color-label :text="$t('schools_page.most_effective_computer_science_solution')" />
           </h1>
           <p class="text-24">
-            {{ $t('schools_page.our_comprehensive_implementation') }}
+            {{ $t('schools_page.start_teach_trial') }}
           </p>
           <CTAButton
-            href="https://codecombat.com/teachers/quote"
-            :description="$t('schools_page.let_us_craft_a_custom_program')"
+            class="contact-solution"
+            @clickedCTA="showContactModal = true"
```

Comment:
Why do we need @clickedCTA? @click doesn't work?

Reply:
Is a click emitted automatically?

I added this in CTAButton: `@click="$emit('clickedCTA')"` and thus used clickedCTA. I can change to use @click if it emitted automatically. Or am I missing something here?

Reply:
right, `@click` will be triggered on description as well.